src/agent/onefuzz-agent/src/tasks/fuzz/libfuzzer_fuzz.rs

```diff
@@ -61,7 +65,7 @@ pub struct Config {
     #[serde(default = "default_bool_true")]
     pub check_fuzzer_help: bool,
 
-    #[serde(default = "default_bool_true")]
+    #[serde(default = "default_bool_false")]
```

Comment:
If you like, this could be:
```suggestion
    #[serde(default)]
```